ci/prow/templates/prow_config_header.yaml

```diff
@@ -100,6 +100,7 @@ tide:
     - do-not-merge/invalid-owners-file
   merge_method:
     knative: squash
+    google: squash
```

Comment:
If `google/knative-gcp` can be used instead, it would be a better choice.

Reply:
done